### ISSUE TITLE
Provide a way to attach pdb to static libraries

### ISSUE THREAD

**Issue by Thieum** [coapp/coapp.powershell#110]
Pdb for static libraries are different than those to provide with dynamic libraries. In the latest dev version, the pdb set in the symbols is copied with the dll. 

Please add a lib-symbols label in the autopkg that allows to provide a pdb for static libraries.

This pdb should be copied next to the lib in the resulting package.

**Comment by luke727** [comment]
Yes, please.  This is screwing up our ability to link with debugging information to static libraries.

**Comment by zabulus** [comment]
You can build .lib files with symbols embedded.
Set /Z7 compatible flag in C/C++ -> General -> Debug Information Format field

**Comment by Thieum** [comment]
Another workaround is to copy the pdb at the right place in the nupkg after its creation with autopkg. This way it won't corrupt the msbuild addons. They will only be unzipped at the right place.
